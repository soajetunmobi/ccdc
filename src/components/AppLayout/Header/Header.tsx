import React, { FunctionComponent } from 'react'
import { routes } from 'const/routes'
import { useHistory } from 'react-router-dom'

export const Header: FunctionComponent = () => {
  const history = useHistory()

  return (
    <div className='header'>
      <div className='logo'>
        <span
          className='cursor--pointer'
          data-testid='logo-link'
          onClick={() => history.push(routes.home)}
        >
          CCDC
        </span>
      </div>
    </div>
  )
}
