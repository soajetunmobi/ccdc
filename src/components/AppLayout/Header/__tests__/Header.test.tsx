import React from 'react'
import { Header } from '../Header'
import { render } from '@testing-library/react'

it('renders Header correctly', () => {
  const { container } = render(<Header />)
  expect(container).toMatchSnapshot()
})
