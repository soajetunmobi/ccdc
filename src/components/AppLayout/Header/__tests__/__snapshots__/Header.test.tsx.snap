// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header correctly 1`] = `
<div>
  <div
    class="header"
  >
    <div
      class="logo"
    >
      <span
        class="cursor--pointer"
        data-testid="logo-link"
      >
        CCDC
      </span>
    </div>
  </div>
</div>
`;
